src/containerapp/azext_containerapp/tests/latest/test_containerapp_validators.py

```diff
@@ -1,29 +0,0 @@
-# --------------------------------------------------------------------------------------------
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for license information.
-# --------------------------------------------------------------------------------------------
-
-
-from azure.cli.testsdk import (ScenarioTest)
-from ..._validators import validate_revision_suffix, ValidationError
-
-class ContainerappRevisionSuffixTests(ScenarioTest):
```

Comment:
test is moved?

Reply:
It is a unit test, test `validate_revision_suffix`, which has been move to Core. The test is exists in Core.